neofetch

```diff
@@ -927,29 +927,20 @@ get_gpu() {
             if [[ -f "/tmp/neofetch/gpu" ]]; then
                 source "/tmp/neofetch/gpu"
             else
-                bdf_number="$(PATH="/sbin:$PATH" lspci -k | grep -A2 -F 'VGA' | grep -B2 -F 'Kernel driver in use' | awk '/^\w/ {print $1}')"
-
-                if [[ -z "$bdf_number" ]]; then
-                    # Fallback if no kernel driver is in use
-                    gpu="$(PATH="/sbin:$PATH" lspci -mm | awk -F '\\"|\\" \\"' '/3D|VGA/ {print $3 " " $4}')"
-
-                else
-                    # Find the currently used GPU by its BDF
-                    gpu="$(PATH="/sbin:$PATH" lspci -mm | awk -v bdf_number="$bdf_number" -F '\\"|\\" \\"' '$0 ~ bdf_number {print $3 " " $4}')"
-                fi
+                gpu="$(PATH="/sbin:$PATH" lspci -mm | awk -F '\\"|\\" \\"' '/3D|VGA/ {print $3 " " $4}')"
```

Comment:
In some devices (including mine), `3D` is replaced by `Display`. Changing `/3D|VGA/` to `/Display|3D|VGA/` may be beneficial here.